source/reporting/index.html.md.erb

```diff
@@ -149,22 +149,15 @@ An example search request:
 
 Some of the query parameters you can use to search for payments are:
 
-* `reference`
-
-* `email`
-
-* `state`
-
-* `card_brand`
-
+* `reference` - case insensitive, exact match only
+* `email` - case insensitive
+* `card_brand` - case insensitive, exact match only
+* `cardholder_name` - case insensitive
+* `state` - case sensitive, exact match only
 * `first_digits_card_number`
-
 * `last_digits_card_number`
```

Comment:
Might want to mention these are both exact matches too?

Reply:
Good call - fixed in https://github.com/alphagov/pay-tech-docs/pull/320/commits/dfcde668ac365d847c513302e39b0f61b8c78056.